game/src/js/scenes/TitleScene.js

```diff
@@ -1,44 +1,60 @@
 import ServiceWorker from 'util/ServiceWorker';
 import Checkbox from 'objects/ui/Checkbox';
+import Button from 'objects/ui/Button'
+
 
 export default class TitleScene extends Phaser.Scene {
 
   constructor() {
     super({key: 'TitleScene'});
   }
-
+  
   preload() {
   }
-    
+
   create() {
-    this.add.image(400, 300, 'sky');
+    this.add.image(400, 430, 'sky');
 
-    let particles = this.add.particles('red');
+    // let particles = this.add.particles('red');
 
-    let emitter = particles.createEmitter({
-      speed: 100,
-      scale: { start: 1, end: 0 },
-      blendMode: 'ADD'
-    });
+    // let emitter = particles.createEmitter({
+    //   speed: 100,
+    //   scale: { start: 1, end: 0 },
+    //   blendMode: 'ADD'
+    // });
 
     let logo = this.add.image(400, 100, 'logo');
```

Comment:
Woohoo! "Ragin' Mages"!!!! We should probably close the game name issue.

Maybe this title should be a bit larger in relation to the buttons. I wonder how it would look with stylized fonts (see comment below) instead of an image. That way we could resize it dynamically.